eventmesh-connectors/eventmesh-connector-http/src/main/java/org/apache/eventmesh/connector/http/sink/handle/WebhookHttpSinkHandler.java

```diff
@@ -83,20 +83,28 @@ private void doInitCallbackServer() {
             .path(this.webhookConfig.getCallbackPath())
             .method(HttpMethod.POST)
             .produces("application/json")
-            .handler(ctx -> {
-                JSONObject callbackData = JSON.parseObject(ctx.body().asString());
-                // store callback data
-                if (!this.callbackQueue.offer(callbackData)) {
-                    log.error("Callback data is full, discard the data. Data: {}", callbackData);
+            .handler(ctx -> ctx.request().body().onComplete(ar -> {
+                if (ar.succeeded()) {
+                    JSONObject callbackData = JSON.parseObject(ar.result().toString());
+                    // store callback data
+                    if (!this.callbackQueue.offer(callbackData)) {
+                        log.error("Callback data is full, discard the data. Data: {}", callbackData);
+                    } else {
+                        log.debug("Succeed to store callback data. Data: {}", callbackData);
+                    }
+                    // response 200 OK
+                    ctx.response()
+                        .putHeader(HttpHeaders.CONTENT_TYPE, "application/json; charset=utf-8")
```

Comment:
We can add a User-Agent header to ensure that our traffic flows smoothly over the Internet.